### ISSUE TITLE
CSV files failed to be previewed by Quicklook in Windows 7

### ISSUE THREAD

**Issue by Sun-Pengcheng** [QL-Win/QuickLook#746]
**Describe the bug**
The csv files failed to be previewed by Quicklook.

**To Reproduce**
 I tried many times, but found all the csv files failed to be previewed. I have used both the  v3.6.9 version and v3.6.10 version, but they all failed. 
**Expected behavior**

**Desktop (please complete the following information):**
 - OS Version: Windows 7 64bit
 - QuickLook Version: [v3.6.9/10] with office plugin

Thanks for the developer very much! 
**Screenshots and Exception Log**
![image](https://user-images.githubusercontent.com/62691819/94436845-879cf280-01cf-11eb-8237-405b37f4cee7.png)
(https://github.com/QL-Win/QuickLook/wiki/Differences-Between-Distributions#user-data-location).

**Comment by rabelux** [comment]
I'm also on win7 64bit and for me csv works fine.
@xupefei are u checking formatting or delimiter before trying to preview the file?

**Comment by Sun-Pengcheng** [comment]
> I'm also on win7 64bit and for me csv works fine.
> @xupefei are u checking formatting or delimiter before trying to preview the file?

Thank you. I am really confused about this. And if I changed the file type to txt or xlsx by renaming the file extention, Quicklook could preview it well.

**Comment by rabelux** [comment]
Would you mind sharing the file with us?

**Comment by Sun-Pengcheng** [comment]
> Would you mind sharing the file with us?
OK, Thanks a lot. It's more likely there are some conflicts in my computer. Because all the csv files I had tried failed to be previewed. I have uploaded one of them as the attachment. I renamed the .csv as .txt because csvs couldn't be uploaded. 
[
[Compare_Characteristic_compare.txt](https://github.com/QL-Win/QuickLook/files/5296119/Compare_Characteristic_compare.txt)

](url)

**Comment by rabelux** [comment]
![grafik](https://user-images.githubusercontent.com/16120882/94513365-d1f99000-021e-11eb-9875-7b45ebac7172.png)
still works for me

**Comment by Sun-Pengcheng** [comment]
> ![grafik](https://user-images.githubusercontent.com/16120882/94513365-d1f99000-021e-11eb-9875-7b45ebac7172.png)
> still works for me

Thanks you very much for your help! Just now, I tried all the latest versions and found an interesting thing. The lower versions (3.6.4/3.6.2/0.3.6.1/0.3.6) successfully previewed the csv files on my computer. But the versions from 3.6.5 to 3.6.10 all failed. Maybe the bug is made from the version upgrade between 3.6.4 and 3.6.5.

**Comment by xupefei** [comment]
Have you checked the CSV plugin do exist? It's at `QuickLook.Plugins\xxxx.CsvViewer\xxxxx.CsvViewer.dll`

**Comment by Sun-Pengcheng** [comment]
> Have you checked the CSV plugin do exist? It's at `QuickLook.Plugins\xxxx.CsvViewer\xxxxx.CsvViewer.dll`

Thank you very much for advice and it has enlighted me. I checked the CSV plugin and it was OK. As I found 3.6.4 version worked well yesterday, I just copied the CSV plugin from 3.6.4 to 3.6.10, and overwrited two files. After reopening Quicklook 3.6.10, I'm excited to find it can preview CSV files! 
![Snipaste_2020-09-30_10-28-04](https://user-images.githubusercontent.com/62691819/94637002-106b7980-0309-11eb-937e-90fbfefebfa0.png)
![Snipaste_2020-09-30_10-28-32](https://user-images.githubusercontent.com/62691819/94637010-12353d00-0309-11eb-9595-6f65f551c02f.png)
![Snipaste_2020-09-30_10-36-57](https://user-images.githubusercontent.com/62691819/94637012-12cdd380-0309-11eb-9776-0aa99dc7fa8f.png)
